X16 Reference - 02 - Getting Started.md

```diff
@@ -192,7 +192,7 @@ BOOT
 
 ### F-KEYS**
```

Comment:
Is this bold to header gone wrong? Couldn't find any significance in the trailing asterisks.